code/mob/living/critter/ai/flock/flocktasks.dm

```diff
@@ -348,17 +341,15 @@ stare
 		if(istype(reserved) && !isfeathertile(reserved))
 			. = get_path_to(holder.owner, reserved, max_dist, 1)
 			if(length(.))
-				return
+				return reserved //TODO make this better
 			else
 				//unreserve the turf if we can't get at it
 				F.flock.busy_tiles[F.real_name] = null
 
 		// if there's a priority tile we can go for, do it
 		var/list/priority_turfs = F.flock.getPriorityTurfs(F)
 		if(length(priority_turfs))
-			. = get_path_to(holder.owner, priority_turfs, max_dist, 1)
-			if(length(.))
-				return
+			return priority_turfs
```

Comment:
won't this mean if there are priority turfs but they're unreachable it will never try to convert other turfs?

Reply:
yes :( I will fix these, I'm not happy with how we were doing that anyway.